cost/azure/unattached_volumes/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure Unused Volumes
+
+## What it does
+
+This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. Using log analytics, we will determine the number of days the disk has been ditached. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
```

Comment:
please make sure we are using the correct text here and spelling is checked.

Reply:
Required changes are done